internal/handlers/pg/fetch.go

```diff
@@ -73,3 +73,37 @@ func nextRow(rows pgx.Rows) (*types.Document, error) {
 
 	return doc.(*types.Document), nil
 }
+
+func (h *Handler) protoFetch(ctx context.Context, param fetchParam) ([]*types.Document, error) {
+	var sql string
+	if param.comment != "" {
+		param.comment = fmt.Sprintf("/* %s */", param.comment)
```

Comment:
From ticket description:

> To prevent SQL injection, we should replace occurrences of /* and */ in the comment with / * and * / (add extra space).

See also [there](https://xkcd.com/327/)